src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -261,6 +262,7 @@ def launch_fluent(
     The allocated machines and core counts are queried from the scheduler environment and
     passed to Fluent.
     """
+    _check_license()
```

Comment:
Looks good.

Reply:
Thank you.

Reply:
@hpohekar We shouldn't do any preemptive checks of this type. These are not in our domain. If the user is allowed to run Fluent, it will run. None of our business. 

If we can diagnose any issues after the fact then at that point we can provide information to the `launch_fluent()` caller (by augmenting exception information).

Reply:
Yes, I agree.